### PR TITLE
Fix bugs on home screen

### PR DESCRIPTION
**Pull request by dishanest**
## Overview

Fix bouncing bug caused by Skeleton animation, update Magazines objects to match new schema, and improve weekly debrief expiration checks. 

## Test Coverage

Playtesting. 

## Related PRs or Issues

@vianxnguyen @jjennifergu please rebase to include this commit before proceeding w/ magazine PDF viewer

## Screenshots

<details>

  <summary>Bouncing bug</summary>

  <table>
  <tr>
     <td>Before</td>
     <td>After</td>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
 </table>
  

</details>

### CODE REVIEW

File: Volume/Models/WeeklyDebrief.swift [hunk 1]
```diff
@@ -26,4 +26,9 @@ struct WeeklyDebrief: Codable {
         readArticleIDs = weeklyDebrief.readArticles.map(\.fragments.articleFields.id)
         randomArticleIDs = weeklyDebrief.randomArticles.map(\.fragments.articleFields.id)
     }
+
+    var isExpired: Bool {
+        let comparisonResult = Calendar.current.compare(self.expirationDate, to: Date.now, toGranularity: .day)
```
Comment (ReadeP) [inline Volume/Models/WeeklyDebrief.swift:31]: Do you need `self` here

File: Volume/View Models/MagazineCell.swift [hunk 2]
```diff
@@ -17,10 +17,12 @@ struct MagazineCell: View {
 
     var body: some View {
         VStack(alignment: .leading) {
-            WebImage(url: magazine.magazineUrl)
-                .resizable()
-                .grayBackground()
-                .aspectRatio(contentMode: .fill)
+            // TODO: figure out how to obtain cover image from PDF
+            SkeletonView()
+//            WebImage(url: magazine.magazineUrl)
```
Comment (ReadeP) [inline Volume/View Models/MagazineCell.swift:22]: delete commented code

**Review by ReadeP — APPROVED** [review]
LGTM

File: Volume/View Models/MagazineCell.swift (hunk 2 above)
Comment (dishanest) [reply Volume/View Models/MagazineCell.swift:22]: Keeping the TODO for jennifer

**Pull request merged**